### DISCUSSION TITLE
Reduce navigation sensitivity on stop

### DISCUSSION THREAD

**Discussion by webnetxpress** [osmandapp/OsmAnd#13710] [Q&A] [answered]
Is there a way to reduce the compass/navigation sensitivity when you slow or come to a stop?  When using this on a motorcycle with map setting set to DIrection of Movement...  When moving the map and direction work perfectly.  However as soon as I come to a stop the map will begin to rotate constantly while I am standing still.  You lose your orientation a lot of times unless you are watching the map before stopping. I'd like to reduce the sensitivity or add a buffer time to keep the map in the same orientation for X seconds before it adjusts.

**Comment by pebogufi** [comment, marked as answer]
Just try that:
Open your pofile, then 
Configure profile | Navigation settings | Map during navigation | Map orientation threshold

It avoids this rotating.
I use a value of 0, test it (may be with different values).

**Comment by webnetxpress** [comment]
thank you! I will try it
